tests/test_operations/test_base.py

```diff
@@ -190,14 +196,61 @@ def test_matrix_repr(self, op, matrix):
     def test_controlled_matrix_repr(self, op, qubits, matrix, two_qubit_circuit):
         """Test that controlled matrix representations are correct."""
         two_qubit_circuit.unlock()
-        with two_qubit_circuit.context as q:
+        with two_qubit_circuit.context as (q, c):
             assert np.allclose(op(*[q[i] for i in qubits]).matrix, matrix)
 
 
-@pytest.mark.xfail(reason="Measurements are not implemented yet.")
 class TestMeasurement:
     """Unit tests for the ``Measurement`` class."""
 
+    def test_initialize_measurement(self):
+        """Test initializing a measurement operation."""
+        m = Measurement()
+        assert m.__repr__() == "<Measurement, qubits=None, measured=None>"
```

Comment:
```suggestion
        assert repr(m) == "<Measurement, qubits=None, measured=None>"
```

Reply:
I have a tendency to use `__repr__` instead of `repr` in test. It somehow _feels_ closer to testing the actual method. I'm not sure if there actually is a difference though, but I'd suspect not.

Reply:
Though the user will be using `repr(...)` so that's what you want to be testing?

Reply:
Sounds reasonable, but `repr()` feels closer to user code :smile:

Reply:
Damn GitHub, not refreshing comments like this :confused: